haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Compat.hs

```diff
@@ -318,6 +330,7 @@ pattern HsForAllTyCompat binders <- HsForAllTy binders _
 pattern HsForAllTyCompat binders <- HsForAllTy _ binders _
 #endif
 
+{-# COMPLETE UserTyVarCompat, KindedTyVarCompat #-}
```

Comment:
I'm not sure about this one because there is actually another case in the datatype for 8.6 (XTyVarBndr)
http://hackage.haskell.org/package/ghc-8.6.5/docs/HsTypes.html#t:HsTyVarBndr

Though I'm not sure whether it's ever used in practice.  The internals of GHC seem to call panic when they encounter it anyway...

Reply:
The similar issue as above.

Reply:
Fixed in a similar way.